sklearn/decomposition/tests/test_factor_analysis.py

```diff
@@ -83,3 +85,31 @@ def test_factor_analysis():
         precision = fa.get_precision()
         assert_array_almost_equal(np.dot(cov, precision),
                                   np.eye(X.shape[1]), 12)
+
+    # test rotation
+    n_components = 2
+
+    results = {}
+    for method in (None, "varimax", 'quartimax'):
+        fa_var = FactorAnalysis(n_components=n_components,
+                                rotation=method)
+        results[method] = fa_var.fit_transform(X)
+    for rot1, rot2 in combinations([None, 'varimax', 'quartimax'], 2):
+        assert not np.allclose(results[rot1], results[rot2])
```

Comment:
Despite being different, is there a way to show that these are equivalent for projection (if that is the case)?

Reply:
I don't think they should be ... but I'll look at something that at least comes close.